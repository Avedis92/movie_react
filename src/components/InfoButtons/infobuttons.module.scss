@import "../sasscommonmixxins/button.scss";
@import "../sasscommonmixxins/flex.scss";
@import "../sasscommonmixxins/mediaqueries.scss";

.movieInfoButtons {
  height: 100%;
  width: 30%;
  :nth-child(1) {
    @include styleButtons(
      100%,
      100%,
      white,
      rgba(0, 0, 0, 0.63),
      10px,
      1px solid white
    );
    cursor: pointer;
    transition: 500ms;
    &:hover {
      transform: scale(105%);
    }
  }
  @include mobile {
    width: 40%;
  }
}
