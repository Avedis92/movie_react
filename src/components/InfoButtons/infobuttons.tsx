import styles from './infobuttons.module.scss';

function InfoButtons({ moreInfo }:{moreInfo:()=>void}) {
  const { movieInfoButtons } = styles;

  return (
    <div className={movieInfoButtons}>
      <button name="moreinfo" type="button" onClick={() => moreInfo()}>More Info</button>
    </div>
  );
}
export default InfoButtons;
