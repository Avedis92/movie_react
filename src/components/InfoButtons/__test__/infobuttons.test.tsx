import { render } from '@testing-library/react';
import InfoButtons from '../infobuttons';

test('Make a snapshot test', () => {
  const {asFragment} = render(<InfoButtons moreInfo={() => null} />);
  expect(asFragment()).toMatchSnapshot();
});
