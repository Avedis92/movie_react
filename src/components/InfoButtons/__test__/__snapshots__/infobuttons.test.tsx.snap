// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make a snapshot test 1`] = `
<DocumentFragment>
  <div
    class="movieInfoButtons"
  >
    <button
      name="moreinfo"
      type="button"
    >
      More Info
    </button>
  </div>
</DocumentFragment>
`;
